src/vs/workbench/contrib/chat/browser/chatQuick.ts

```diff
@@ -173,14 +175,24 @@ class QuickChat extends Disposable {
 			});
 		}
 
-		this.registerListeners();
+		this.sash?.dispose();
+		this.sash = this._register(new Sash(parent, { getHorizontalSashTop: () => parent.offsetHeight }, { orientation: Orientation.HORIZONTAL }));
+		this.registerListeners(parent);
 	}
 
-	private registerListeners(): void {
+	private registerListeners(parent: HTMLElement): void {
 		this._register(this.widget.inputEditor.onDidChangeModelContent((e) => {
 			this._currentQuery = this.widget.inputEditor.getValue();
 		}));
 		this._register(this.widget.onDidClear(() => this.clear()));
+		this._register(this.sash.onDidChange((e) => {
+			if (e.currentY < 200) {
+				return;
+			}
+			this.widget.layout(e.currentY, parent.offsetWidth);
```

Comment:
Same comment with this width

Reply:
Fixed that in https://github.com/microsoft/vscode/pull/191431